client/src/app/shared/services/alert.service.ts

```diff
@@ -50,6 +50,6 @@ export class AlertService {
 
   clear() {
     // clear alerts
```

Comment:
Why not rename the function and remove the comment?

Reply:
I think this is out of scope for this PR

Reply:
Done but please, let's just use a code formatter. These kind of comments don't need to be part of a PR

Reply:
but done